src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/Microsoft.NET.Sdk.WebAssembly.Pack.pkgproj

```diff
@@ -0,0 +1,17 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <PropertyGroup>
+    <PackageDescription>SDK for building and publishing WebAssembly applications.</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(RepoTasksDir)Microsoft.NET.Sdk.WebAssembly.Pack.Tasks\Microsoft.NET.Sdk.WebAssembly.Pack.Tasks.csproj" />
+
+    <PackageFile Include="build\*.props;build\*.targets;build\*.web.config" TargetPath="build" />
+    <PackageFile Include="$(RepoRoot)artifacts\bin\Microsoft.NET.Sdk.WebAssembly.Pack.Tasks\$(Configuration)\net472\*.dll" TargetPath="tools\net472" />
+    <PackageFile Include="$(RepoRoot)artifacts\bin\Microsoft.NET.Sdk.WebAssembly.Pack.Tasks\$(Configuration)\net8.0\*.dll" TargetPath="tools\net8.0" />
```

Comment:
@steveisok Is there a nicer way to include outputs from a project reference? I tried adding `Pack="true" PackagePath="tools"`, but it doesn't do anything.

Reply:
@steveisok ?

Reply:
@steveisok I had to move the `PackageFile` to a target, because at the evalution phase, the dlls don't exist yet. I would love to have a better way to include `ProjectReference` outputs in the nupkg.

Reply:
@radical Any thoughts☝️?

Reply:
Yep, like @steveisok . The targets for pkgproj will invoke `GetFilesToPackage` target on the project references, so you can surface the files from those projects.

Reply:
Thanks!